views/assets/src/tailwind.css

```
@@ -626,6 +646,39 @@ ol[data-sonner-toaster],
+[data-sonner-toast] {
+  overflow: hidden;
+}
+[data-sonner-toast]::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  transform-origin: left center;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)